include/swift/Basic/SourceLoc.h

```diff
@@ -17,6 +17,8 @@
 #ifndef SWIFT_BASIC_SOURCELOC_H
 #define SWIFT_BASIC_SOURCELOC_H
 
+#include <assert.h>
```

Comment:
I've moved it down, is that the location you were after?

(I only put it up there because I thought the standard headers come first, I'm not sure if there's a header order standard for the project.)

Reply:
Yeap, thanks.
I don't think Swift has its own `#include` conventions, and the [LLVM conventions](https://llvm.org/docs/CodingStandards.html#include-style) say that the system includes go last.